openpype/hosts/aftereffects/plugins/create/create_render.py

```diff
@@ -23,12 +24,13 @@ def __init__(self, project_settings, *args, **kwargs):
                                                   ["create"]
                                                   ["RenderCreator"]
                                                   ["defaults"])
+        self._host = AfterEffectsHost()
```

Comment:
Don't create new instance of object. BTW creators have access to `self.host`, you don't have to create it at all.